appveyor.yml

```diff
@@ -57,19 +57,22 @@ install:
   # Update pip/setuptools/wheel to avoid an issue like
```

Comment:
There's also `.gitlab-ci.yml`, which is being used [here](https://gitlab.tiker.net/inducer/relate). Just invited you to that project. Could you also hack that into shape to use poetry?